edge-agent-sdk/src/commonMain/kotlin/org/hyperledger/identus/walletsdk/apollo/ApolloImpl.kt

```diff
@@ -40,11 +40,16 @@ import org.hyperledger.identus.walletsdk.domain.models.keyManagement.SeedKey
 import org.hyperledger.identus.walletsdk.domain.models.keyManagement.StorableKey
 import org.hyperledger.identus.walletsdk.domain.models.keyManagement.StorablePrivateKey
 import org.hyperledger.identus.walletsdk.domain.models.keyManagement.TypeKey
+import org.hyperledger.identus.walletsdk.logger.EdgeAgentLogger
+import org.hyperledger.identus.walletsdk.logger.EdgeAgentLoggerImpl
+import org.hyperledger.identus.walletsdk.logger.LogComponent
 
 /**
  * Apollo defines the set of cryptographic operations that are used in the Atala PRISM.
  */
-class ApolloImpl : Apollo {
+class ApolloImpl(
+    private val logger: EdgeAgentLogger = EdgeAgentLoggerImpl(LogComponent.APOLLO)
```

Comment:
really minor, but I find the naming a bit odd, `EdgeAgentLogger` creates coupling to the EdgeAgent which is a separate component. I'd expect a Logger to have a meaningful name related to how it logs, like "ConsoleLogger" or "FileLogger" ...

Reply:
That is a very good point. I will rename it.